### ISSUE TITLE
When its is used for Production ready?

### ISSUE THREAD

**Issue by joshua750** [komalsdagde/document_viewer#6]
Hello please tell me when this package is ready for production App. Also tell me its comply all google play policies?

**Comment by komalsdagde** [comment]
Yes, you can use the app. If you encounter any issues, please let me know.
